frappe/workflow/doctype/workflow_action/workflow_action.py

```diff
@@ -140,14 +140,23 @@ def update_completed_workflow_actions(doc, user=None):
 
 def get_next_possible_transitions(workflow_name, state):
 	return frappe.get_all('Workflow Transition',
-		fields=['allowed', 'action', 'state', 'allow_self_approval'],
+		fields=['allowed', 'action', 'state', 'allow_self_approval', \
+			"email_based_on", "docfield_name", "user"],
 		filters=[['parent', '=', workflow_name],
 		['state', '=', state]])
 
 def get_users_next_action_data(transitions, doc):
 	user_data_map = {}
 	for transition in transitions:
-		users = get_users_with_role(transition.allowed)
```

Comment:
needs test coverage

Reply:
Added test coverage and tested it

Reply:
@navdeepghai1 every change in now *requires* a test case (even if there were no cases earlier). There will be tests in **Workflow**. Please make sure your features have test coverage to ensure they work and also are safe from any future breaking changes.